sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -2019,6 +2019,75 @@ def get_page_ranges( # type: ignore
             process_storage_error(error)
         return get_page_ranges_result(ranges)
 
+    @distributed_trace
+    def get_managed_disk_page_range_diff( # type: ignore
```

Comment:
The decision (after the last discussion) is to add a parameter to the existing page diff API.

Reply:
It seems the other ask separate the disk diff method into a different one https://github.com/Azure/azure-sdk-for-net/pull/9195/files

Reply:
My bad, the decision was reverted. 🤣